xmodule/tests/test_annotatable_block.py

```diff
@@ -134,3 +134,11 @@ def test_extract_instructions(self):
         xmltree = etree.fromstring('<annotatable>foo</annotatable>')
         actual = self.annotatable._extract_instructions(xmltree)  # lint-amnesty, pylint: disable=protected-access
         assert actual is None
+
+    def test_instruction_removal(self):
+        xmltree = etree.fromstring(self.sample_xml)
+        instructions = self.annotatable._extract_instructions(xmltree)  # lint-amnesty, pylint: disable=protected-access
```

Comment:
```suggestion
        instructions = self.annotatable._extract_instructions(xmltree)  # pylint: disable=protected-access
```

`lint-amnesty` is only for pylint violations which were [grandfathered in](https://docs.openedx.org/projects/openedx-proposals/en/latest/archived/oep-0034-bp-lint-amnesty.html). You should not put that label on new violations.